docs/reference/buildx_create_for_multi_arch_build_in_k8s.md

```diff
@@ -0,0 +1,74 @@
+# Buildx create for multi-arch builds in k8s
+
+## With QEMU emulation
+
+```console
+$ KUBECONFIG=${KUBECONFIG} \
+    docker buildx create \
+        --name=builder \
+        --platform=linux/amd64,linux/arm64 \
+        --driver=kubernetes \
+        --driver-opt=namespace=buildkit,withqemu=true
+```
+
+### Known Issues
+
+QEUM only work well for `executing`, but for `compiling`, it will be very slow and may be panic.
```

Comment:
- typo: QEUM

- No need to quote "executing" and "compiling"

- What do you mean by "panic"?
-

Reply:
> Slowness is not specific to compilation

Slowness for executing could be acceptable, but for compiling, obviously slow.

> What do you mean by "panic"?

throw some error like `qemu: uncaught target signal 11 (Segmentation fault) - core dumped`
https://github.com/docker/buildx/issues/516#issuecomment-781685997

Maybe rename to `segmentation fault`?